ui/raidboss/data/07-dt/raid/r6s.ts

```diff
@@ -72,13 +94,47 @@ const triggerSet: TriggerSet<Data> = {
       id: 'R6S Sticky Mousse',
       type: 'StartsUsing',
       netRegex: { id: 'A695', source: 'Sugar Riot', capture: false },
-      response: Responses.spread(),
+      response: Responses.spreadThenStack(),
+    },
+    {
+      id: 'R6S Color Riot Debuff Tracker',
+      type: 'GainsEffect',
+      netRegex: { effectId: ['1163', '1164'], capture: true },
+      condition: Conditions.targetIsYou(),
+      run: (data, matches) => data.colorRiotTint = matches.effectId === '1163' ? 'warm' : 'cool',
```

Comment:
It took me a few seconds to realize why this property was phrased this way, but it sure does make sense!

Reply:
The statuses you get are `Warm Tint` and `Cool Tint`, that's why I named it this way.